### PR TITLE
i18n(es): Add `appwriteio.mdx`

### PR DESCRIPTION
**Pull request by AitorMT**
#### Description (required)

The `appwriteio` page has been added.

### CODE REVIEW

**Review by dreyfus92 — APPROVED** [review]
LGTM ✅

File: src/content/docs/es/guides/backend/appwriteio.mdx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+title: Appwrite y Astro
+description: Añade un backend a tu proyecto con Appwrite
+type: backend
+service: Appwrite
+stub: true
```
Comment (Waxer59) [inline src/content/docs/es/guides/backend/appwriteio.mdx:6]:
```suggestion
stub: true
i18nReady: true
```

**Review by Waxer59 — COMMENTED** [review]
Buena PR 🤠, te dejo una sugerencia!

**Review by dreyfus92 — APPROVED** [review]
Gracias @AitorMT y @Waxer59 por revisar 😁

**Pull request merged**